GPTResponder.py

```diff
@@ -59,13 +69,16 @@ def generate_response_from_transcript(self, transcript):
         return self.generate_response_from_transcript_no_check(transcript)
 
     def respond_to_transcriber(self, transcriber):
+        """Thread method to continously update the transcript
+        """
         while True:
 
             if transcriber.transcript_changed_event.is_set():
                 start_time = time.time()
 
                 transcriber.transcript_changed_event.clear()
-                transcript_string = transcriber.get_transcript(length=MAX_PHRASES)
+                transcript_string = transcriber.get_transcript(
```

Comment:
There are 2 other places where MAX_PHRASES is used. Should replace all usage by the new constant and remove MAX_PHRASES completely.

Reply:
Good suggestion. Changed as requested.